ncl.md

```diff
@@ -1,4 +1,4 @@
-# Network Commons License (v1.0.4)
+# Network Commons License
```

Comment:
I suggest we keep the version number if we're not changing anything. It will be easier to adopt future versions if we know where we start.